UI.AutomationTests/Admin/Booking/CreatingMultiDayHearingIndividualDateTests.cs

```diff
@@ -0,0 +1,90 @@
+namespace UI.AutomationTests.Admin.Booking;
+
+public class CreatingMultiDayHearingIndividualDateTests : AdminWebUiTest
+{
+    private BookingDto _bookingDto;
+    private string _groupId;
+
+    [Category("Daily")]
```

Comment:
if all tests are to be marked as daily, we may as well update the test category filter on the daily pipeline

Reply:
Yeah I don't know what the criteria is for what should/shouldn't be a daily test to be honest. But I agree the attribute isn't really needed at the moment